CHANGES

```diff
@@ -18,9 +25,11 @@ PyVISA Changelog
     have been removed
   - `p.u.get_shared_library_arch` now returns `p.u.PEMachineType` instead of a string
   - `p.u.get_arch` now returns a list of `p.u.ArchitectureType`
-- update `Resource` context manager type annotation to preserve derived type PR #740
 - added support for using custom simulation files when using the `sim` backend
   with `pyvisa-shell`. PR #774
+- update `Resource` context manager type annotation to preserve derived type PR # 740
+- added support for an optional monitoring interface to the read_binary_values() and
+    query_binary_values() methods of message-based resources.
```

Comment:
Should be reverted.

Reply:
Done (mostly).  Lines 31 and 32 above are still removed because the optional monitoring interface was not added to  version 1.14.0.